common/bytes.go

```diff
@@ -107,14 +107,9 @@ func ReadVarInt(buff []byte) (ret uint64) {
 	return
 }
 
-// Copy bytes
-//
 // Returns an exact copy of the provided bytes
 func CopyBytes(b []byte) (copiedBytes []byte) {
-	copiedBytes = make([]byte, len(b))
-	copy(copiedBytes, b)
-
-	return
+	return append([]byte{}, b...)
```

Comment:
I think it would be more useful to replace uses of CopyBytes across the implementation.
Append to empty slice is shorter than calling the function.

```text
common.CopyBytes(foobar)
append([]byte, foobar)
```